### PR TITLE
Enhancing 'Hetzner example' docs

### PR DESCRIPTION
**Pull request by NicoWde**
after running into the same issue as #50, I hoped this would help :)

Any feedback is appreciated!

### CODE REVIEW

**Comment by xunleii** [comment]
Hi @NicoWde, sorry for the 2 weeks off non response, I had many things to do.
Thanks a lot for your help, I will try to read it this week.

File: examples/hcloud-k3s/README.md [hunk 1]
```diff
@@ -2,9 +2,27 @@
 
 Configuration in this directory creates a k3s cluster resources including network, subnet and instances.
 
+## Preparations
+
+Make sure your [SSH-Agent](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent) is running (it is neccessary for Terraform), and if not, start it and add your ssh-key:
```
Comment (tedsteen) [inline examples/hcloud-k3s/README.md:7]: This is a good one. I always run into this one.

Comment (tedsteen) [reply examples/hcloud-k3s/README.md:7]: But instead of writing an example wouldn't it be better to simply link to some official documentation?

Comment (NicoWde) [reply examples/hcloud-k3s/README.md:7]: I thought the Link to SSH-Agent Docs would be enough. How would you do it? :)

Comment (xunleii) [reply examples/hcloud-k3s/README.md:7]: Sounds great with a documentation link, thanks a lot for your reviews !

**Review by xunleii — APPROVED** [review]

**Review by xunleii — APPROVED** [review]
@tedsteen what do you think about this PR ?

**Comment by tedsteen** [comment]
> @tedsteen what do you think about this PR ?

I think it's fine! I'd personally skip the example (I think it's mostly noise when you want to focus on k3s terraforming) but I totally see you point with copy/pase. You're the owner so you're the boss :)

**Comment by xunleii** [comment]
> I think it's mostly noise when you want to focus on k3s terraforming

I also see your point and I agree with you. I will move this example in the Wiki (in the future) to keep both readability and accuracy.

>  You're the owner so you're the boss :)

You're the end user, you're much more important than the boss ;)

@NicoWde @tedsteen thanks a lot for your work, I really appreciate :)

**Pull request merged**